include/sir/helpers.h

```diff
@@ -29,6 +29,10 @@
 # include "sir/types.h"
 # include "sir/errors.h"
 
+# if defined(SIR_DPSPRINTF)
+#  include "sir/dpsprintf.h"
```

Comment:
Don't need this here; this header includes `sir/types.h` which includes `sir/platform.h` which already has this same block

Reply:
Aight.